charts/base-cluster/templates/monitoring/kube-prometheus-stack/_helpers.yaml

```diff
@@ -1,9 +1,9 @@
 {{- define "base-cluster.monitoring.ingress" -}}
   {{- $ingress := dig .name "ingress" nil .context.Values.monitoring | required (printf "You need to set the `ingress` for %s" .name) -}}
-  {{- if and .context.Values.certManager.email $ingress.enabled (or .context.Values.global.baseDomain $ingress.customDomain) -}}
+  {{- if and .context.Values.certManager.email $ingress.enabled .context.Values.ingress.enabled (or .context.Values.global.baseDomain $ingress.customDomain) -}}
     {{- $host := include (printf "base-cluster.%s.host" .name) .context -}}
 ingress:
-  enabled: true
+  enabled: {{ .context.Values.ingress.enabled }}
```

Comment:
You don't need to do both, and thinking about it, I think I would prefer the check in the `if`